FurCoNZ/Services/Payment/Stripe/StripeHostedService.cs

```diff
@@ -0,0 +1,134 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Microsoft.Extensions.DependencyInjection;
+using Stripe;
+using Stripe.Checkout;
+
+using FurCoNZ.Configuration;
+using FurCoNZ.DAL;
+
+namespace FurCoNZ.Services.Payment.Stripe
+{
+    public class StripeHostedService : IHostedService, IDisposable
+    {
+        private readonly IHostingEnvironment _hostingEnvironment;
+        private readonly IServiceProvider _services;
+        private readonly IOptions<StripeSettings> _options;
+        private readonly ILogger _logger;
+        private readonly EventService _eventService;
+
+        private Timer _timer;
+        private EventListOptions _eventListOptions;
+
+        private CancellationTokenSource _cancellationTokenSource;
+
+        public static TimeSpan PollInterval = TimeSpan.FromMinutes(5);
+
+        public StripeHostedService(IHostingEnvironment hostingEnvironment, IServiceProvider serviceProvider, IOptions<StripeSettings> options, ILogger<StripeHostedService> logger)
+        {
+            _hostingEnvironment = hostingEnvironment;
+            _services = serviceProvider;
+            _options = options;
+            _logger = logger;
+
+            _eventService = new EventService();
+
+            if (_hostingEnvironment.IsDevelopment())
+                PollInterval = TimeSpan.FromSeconds(30);
```

Comment:
Small style suggestion: either keep this on the same line, or use curly braces. Try to avoid single indented lines since these look like method chains.
```suggestion
            {
                PollInterval = TimeSpan.FromSeconds(30);
            }
```